src/main/resources/blockly/js/mcreator_blocks.js

```diff
@@ -513,6 +513,12 @@ Blockly.Extensions.register('effect_list_provider',
             arrayToBlocklyDropDownArray(javabridge.getListOf("effect"))), 'potion'); // field name is potion for legacy reasons
     });
 
+Blockly.Extensions.register('potion_list_provider',

```

Comment:
Any particular reason this was added?

Reply:
Not really, but it can be used by plugins/future PRs

Reply:
Ok, I see